users/models.py

```diff
@@ -378,6 +379,12 @@ def should_skip_onboarding(self):
             or self.courserunenrollment_set(manager="all_objects").exists()
         )
 
+    @property
+    def openedx_user_exists(self):
```

Comment:
I have renamed from `edx_user_exists` to `openedx_user_exists ` and also fixed the usages.